docs/bugs_and_glitches.md

```diff
@@ -628,161 +594,139 @@ CalcExpAtLevel:
 -; catch rate than BRN/PSN/PAR, which in turn provide a higher catch rate than
 -; no status effect at all. But instead, it makes BRN/PSN/PAR provide no
 -; benefit.
 -; Uncomment the line below to fix this.
-	ld b, a
-	ld a, [wEnemyMonStatus]
-	and 1 << FRZ | SLP
-	ld c, 10
-	jr nz, .addstatus
+ 	ld b, a
+ 	ld a, [wEnemyMonStatus]
+ 	and 1 << FRZ | SLP
+ 	ld c, 10
+ 	jr nz, .addstatus
 -	; ld a, [wEnemyMonStatus]
 +	ld a, [wEnemyMonStatus]
-	and a
-	ld c, 5
-	jr nz, .addstatus
-	ld c, 0
-.addstatus
-	ld a, b
-	add c
-	jr nc, .max_1
-	ld a, $ff
-.max_1
+ 	and a
+ 	ld c, 5
+ 	jr nz, .addstatus
+ 	ld c, 0
+ .addstatus
+ 	ld a, b
+ 	add c
+ 	jr nc, .max_1
+ 	ld a, $ff
+ .max_1
 ```
 
 
 ## Moon Ball does not boost catch rate
 
-This is a bug with `MoonBallMultiplier` in [items/item_effects.asm](/items/item_effects.asm):
-
-
-**Fix:**
+**Fix:** Edit `MoonBallMultiplier` in [items/item_effects.asm](/items/item_effects.asm):
 
 ```diff
 -; Moon Stone's constant from Pokémon Red is used.
 -; No Pokémon evolve with Burn Heal,
 -; so Moon Balls always have a catch rate of 1×.
-	push bc
-	ld a, BANK("Evolutions and Attacks")
-	call GetFarByte
+ 	push bc
+ 	ld a, BANK("Evolutions and Attacks")
+ 	call GetFarByte
 -	cp MOON_STONE_RED ; BURN_HEAL
 +	cp MOON_STONE
-	pop bc
-	ret nz
+ 	pop bc
+ 	ret nz
 ```
 
 
 ## Love Ball boosts catch rate for the wrong gender
 
-This is a bug with `LoveBallMultiplier` in [items/item_effects.asm](/items/item_effects.asm):
-
-
-**Fix:**
+**Fix:** Edit `LoveBallMultiplier` in [items/item_effects.asm](/items/item_effects.asm):
 
 ```diff
-.wildmale
+ .wildmale
 
-	ld a, d
-	pop de
-	cp d
-	pop bc
+ 	ld a, d
+ 	pop de
+ 	cp d
+ 	pop bc
 -	ret nz ; for the intended effect, this should be "ret z"
 +	ret z
 ```
 
 
 ## Fast Ball only boosts catch rate for three Pokémon
 
-This is a bug with `FastBallMultiplier` in [items/item_effects.asm](/items/item_effects.asm):
-
-
-**Fix:**
+**Fix:** Edit `FastBallMultiplier` in [items/item_effects.asm](/items/item_effects.asm):
 
 ```diff
-.loop
-	ld a, BANK(FleeMons)
-	call GetFarByte
-
-	inc hl
-	cp -1
-	jr z, .next
-	cp c
+ .loop
+ 	ld a, BANK(FleeMons)
+ 	call GetFarByte
+
+ 	inc hl
+ 	cp -1
+ 	jr z, .next
+ 	cp c
 -	jr nz, .next ; for the intended effect, this should be "jr nz, .loop"
 +	jr nz, .loop
-	sla b
-	jr c, .max
+ 	sla b
+ 	jr c, .max
 ```
 
 
 ## Dragon Scale, not Dragon Fang, boosts Dragon-type moves
 
 *Fixing this bug will break compatibility with standard Pokémon Crystal for link battles.*
 
-This is a bug with `ItemAttributes` in [data/items/attributes.asm](/data/items/attributes.asm):
-
-
-**Fix:**
+**Fix:** Edit `ItemAttributes` in [data/items/attributes.asm](/data/items/attributes.asm):
 
 ```diff
-; DRAGON_FANG
+ ; DRAGON_FANG
 -	item_attribute 100, HELD_NONE, 0, CANT_SELECT, ITEM, ITEMMENU_NOUSE, ITEMMENU_NOUSE
 +	item_attribute 100, HELD_DRAGON_BOOST, 0, CANT_SELECT, ITEM, ITEMMENU_NOUSE, ITEMMENU_NOUSE
-```
-
-And:
-
-```diff
-; DRAGON_SCALE
+ ...
```

Comment:
I don't like dotdotdots... They add very little to the context since the preceding part could be anywhere in the file.
I'd avoid them if possible.

Reply:
I think "`; DRAGON_SCALE`" provides sufficient context here. The alternative would be two separate code blocks, but that looks like you're editing two different files, and clarifying "Edit the same file again" is more wordy than just these ellipses.